src/main/webapp/WEB-INF/pages/register.jsp

```diff
@@ -68,11 +68,9 @@
                     <div class="field">
                         <input type="submit" value="Register">
                     </div>
-<!--                    <div class="signup-link">
-                        Already a member? <a href="/WEB-INF/pages/login.jsp"">Log in</a> <%-- To do for later (if time allows) --%>
-                    </div>-->
                 </form>
             </div>
+            <script src="${pageContext.request.contextPath}/js/validation.js"></script>
```

Comment:
Validation script is already included in `pageTemplate`. No need to add it here as well.

Reply:
I know it is in the page template but this page does not include the navbar so no validation script